.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+relative_files = True
+[xml]
+output = coverage.xml
```

Comment:
missing new line and I think we should use `pyproject.toml` instead of new file.

Reply:
Can we also report an issue about using `branch = True` as we discussed yesterday?